Bottom: c5cdc7682f56697c7748fe6882d4e29f6715d1f1
Top:    cc399ed712c028dc8c1764a3f68860c514c8390f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 20:18:15 +0100

hid/gtk: Use g_signal_connect(), not gtk_signal_connect()

gtk_signal_connect() is deprecated.


---

diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index a4d7b12..0ea661b 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -228,8 +228,7 @@ ghid_check_button_connected (GtkWidget * box,
     gtk_box_pack_end (GTK_BOX (box), b, expand, fill, pad);
 
   if (cb_func)
-    gtk_signal_connect (GTK_OBJECT (b), "clicked",
-			GTK_SIGNAL_FUNC (cb_func), data);
+    g_signal_connect (b, "clicked", G_CALLBACK (cb_func), data);
   if (button)
     *button = b;
 }
@@ -251,8 +250,7 @@ ghid_button_connected (GtkWidget * box, GtkWidget ** button,
     gtk_box_pack_end (GTK_BOX (box), b, expand, fill, pad);
 
   if (cb_func)
-    gtk_signal_connect (GTK_OBJECT (b), "clicked",
-			GTK_SIGNAL_FUNC (cb_func), data);
+    g_signal_connect (b, "clicked", G_CALLBACK (cb_func), data);
   if (button)
     *button = b;
 }
